packages/@react-spectrum/dialog/test/AlertDialog.test.js

```diff
@@ -0,0 +1,141 @@
+/*
+ * Copyright 2020 Adobe. All rights reserved.
+ * This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License. You may obtain a copy
+ * of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under
+ * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ * OF ANY KIND, either express or implied. See the License for the specific language
+ * governing permissions and limitations under the License.
+ */
+
+import {AlertDialog} from '../';
+import {cleanup, render} from '@testing-library/react';
+import React from 'react';
+import {triggerPress} from '@react-spectrum/test-utils';
+
+
+describe('AlertDialog', function () {
+  afterEach(cleanup);
+
+  it('renders alert dialog with onConfirm', function () {
+    let onConfirmSpy = jest.fn();
+    let {getByRole} = render(
+      <AlertDialog variant="confirmation" title="the title" primaryLabel="confirm" onConfirm={onConfirmSpy}>
+        Content body
+      </AlertDialog>
+    );
+
+    let dialog = getByRole('alertdialog');
+    expect(document.activeElement).toBe(dialog);
+
+    let button = getByRole('button');
+    triggerPress(button);
+    expect(onConfirmSpy).toHaveBeenCalledTimes(1);
+    expect(onConfirmSpy).toHaveBeenCalledWith('primary');
+  });
+
+  it('renders alert dialog with onConfirm / onCancel', function () {
+    let onCancelSpy = jest.fn();
+    let onConfirmSpy = jest.fn();
+    let {getByRole, getByText} = render(
+      <AlertDialog variant="confirmation" title="the title" primaryLabel="confirm" cancelLabel="cancel" onConfirm={onConfirmSpy} onCancel={onCancelSpy}>
+        Content body
+      </AlertDialog>
+    );
+
+    let dialog = getByRole('alertdialog');
+    expect(document.activeElement).toBe(dialog);
+
+    let button = getByText('cancel');
+    triggerPress(button);
+    expect(onConfirmSpy).toHaveBeenCalledTimes(0);
+    expect(onCancelSpy).toHaveBeenCalledTimes(1);
+    expect(onCancelSpy).toHaveBeenCalledWith();
+  });
+
+  it('renders alert dialog with onConfirm / onCancel', function () {
```

Comment:
this has the same test description as the previous test, `renders alert dialog with onConfirm / onCancel`.

Also, why aren't we checking what `onConfirmSpy` was called with in the previous test and `onCancelSpy` in this test so we do see that things are different?